content/sensu-go/6.8/operations/deploy-sensu/secure-postgres.md

```diff
@@ -0,0 +1,374 @@
+postgres         | 10.0.0.43   | postgres.example.com              | localhost, 127.0.0.1 |
+
+The additional names for localhost and 127.0.0.1 are added here for convenience and are not strictly required.
+
+- The values provided for the ADDRESS variable will be used to populate the certificate's SAN records.
+For systems with multiple hostnames and IP addresses, add each to the comma-delimited value of the ADDRESS variable.
+- The value provided for the NAME variable will be used to populate the certificate's CN record.
+It will also be used in the names for the `*.pem` and `*.csr` files.
+
+For example:
+
+{{< code shell >}}
+export ADDRESS=localhost,127.0.0.1,10.0.0.43,postgres,postgres.example.com
+export NAME=postgres.example.com
+echo '{"CN":"'$NAME'","hosts":[""],"key":{"algo":"rsa","size":2048}}' | cfssl gencert -config=ca-config.json -profile="postgresql" -ca=ca.pem -ca-key=ca-key.pem -hostname="$ADDRESS" - | cfssljson -bare $NAME
+{{< /code >}}
+
+The `/etc/sensu/tls` directory should now include the following files for your PostgreSQL instance:
+
+ filename        | description |
+-----------------|-------------|
+`postgres.example.com.pem`     | The certificate that your PostgreSQL instance will use.|
+`postgres.example.com-key.pem` | The private key that your PostgreSQL instance will use. |
+`postgres.example.com.csr`     | Certificate signing request for the PostgreSQL certificate. Not used. |
+
+## Generate certificate and key for your Sensu backend
+
+Just like the certificate and key for PostgreSQL, you'll need a certificate and key for the Sensu backend.
+
+To generate the backend certificate and key, run:
+
+{{< code shell >}}
+export POSTGRES_USERNAME=sensu
+echo '{"CN":"'$POSTGRES_USERNAME'","hosts":[""],"key":{"algo":"rsa","size":2048}}' | cfssl gencert -config=ca-config.json -ca=ca.pem -ca-key=ca-key.pem -hostname="" -profile=backend - | cfssljson -bare $POSTGRES_USERNAME
+{{< /code >}}
+
+You should now have the following files in your `/etc/sensu/tls` directory.
+The Sensu backend will use these certificate files to communicate with PostgreSQL:
+
+ filename        | description |
+-----------------|-------------|
+`sensu.pem`      | The certificate that your Sensu backend will use.|
+`sensu-key.pem`  | The private key that your Sensu backend will use. |
+`sensu.csr`      | Certificate signing request for the Sensu backend certificate. Not used. |
+
+Now that you have the required certificates and keys, you can configure Sensu to use certificate authentication with PostgreSQL.
+
+## Configure Sensu to use certificate authentication with PostgreSQL
+
+{{% notice note %}}
+**NOTE**: The Sensu backend uses the libpq library to make connections to PostgreSQL.
+This library [supports a number of environment variables](https://www.postgresql.org/docs/current/libpq-envars.html) that can be injected into the PostgreSQL data source name (DSN) and are loaded at runtime using the system's environment variable file.
+{{% /notice %}}
```

Comment:
Is it possible to expand slightly to explain the significance of this information for people who are using this guide? For example, can they use these other environment variables for more customized configuration? Or are we just explaining where we got the environment variables we're using below?

Reply:
I've included a bit more detail in the message.